workflows/smrnaseq.nf

```diff
@@ -124,26 +125,37 @@ workflow SMRNASEQ {
     // SUBWORKFLOW: Read QC and trim adapters
     //
 
+    if(!params.skip_qc){
     FASTQC_FASTP (
         ch_cat_fastq,
+        ch_fastp_adapters,
         false,
         false
     )
-
     ch_versions = ch_versions.mix(FASTQC_FASTP.out.versions)
-
     reads_for_mirna = FASTQC_FASTP.out.reads
 
+
     //
     // SUBWORKFLOW: mirtrace QC
     //
-    FASTQC_FASTP.out.adapterseq
-    | join( ch_cat_fastq )
-    | map { meta, adapterseq, fastq -> [meta + [adapter:adapterseq], fastq] }
-    | MIRTRACE
+    ch_outputs_for_mirtrace = FASTQC_FASTP.out.adapterseq
+    .join( ch_cat_fastq )
+    .map { meta, adapterseq, fastq -> [meta + [adapter:adapterseq], fastq] }
+    .collect()
+
+    MIRTRACE(ch_outputs_for_mirtrace)
 
     ch_versions = ch_versions.mix(MIRTRACE.out.versions.ifEmpty(null))
 
+    } else{
+        //TODO - rob? :-)
+
+    }
+
+
+
+
```

Comment:
Something is wrong there

Reply:
@apeltzer Can you elaborate?
cf https://github.com/nf-core/smrnaseq/blob/ae02a7607d9aba540e21ec61baaddcba4565623e/workflows/smrnaseq.nf#L147-L153

Reply:
We had some discussions with respect to having individual sample level modes for fastp. Not sure if this was already achieved by updating the module already... 

Not really have time to look after it right now unfortunately 😅